src/devices/formatdevice.py

```diff
@@ -0,0 +1,65 @@
+# Log Parser for RTI Connext.
+#
+#   Copyright 2016 Real-Time Innovations, Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""Device for the output format.
+
+The module contains the abstract class representation for a formatter of the
+parsed logs.
+
+Classes:
```

Comment:
I think that this comment it is not necessary: we only have a class in this file (with the same name of the class).